src/NorthwindWebsite/Business/Models/ChangeRoleModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace NorthwindWebsite.Business.Models;
+
+public class ChangeRoleModel
+{
+    public string UserId { get; set; }
+    public string UserEmail { get; set; }
+    public List<IdentityRole> AllRoles { get; set; }
+    public IList<string> UserRoles { get; set; }
+    public ChangeRoleModel()
+    {
+        AllRoles = new List<IdentityRole>();
```

Comment:
Well, I would say, such way of initialising your collection is a little bit old-fashioned, you could just set inline initialisation